module/observations/helper.js

```diff
@@ -99,6 +102,7 @@ module.exports = class ObservationsHelper {
                     userId,
                     _.omit(data,["entities"]),
                     true,
+                    [],
```

Comment:
@priyanka-TL  This should not be empty array. Do not modify existing code. Check existing code please.